_includes/title/title_header.html

```diff
@@ -3,89 +3,9 @@
-              <div class="c-card-header c-card-header--white">
-              </div>
-              <div class="c-card-image-wrapper">
-                <a href="https://www.pirati.cz/volby/">
-                  <img src="{{ '/assets/img/miscellaneous/volebni-rozcestnik.png' | prepend: site.baseurl }}" class="c-card__image" alt="Černé na bílém">
-                </a>
-              </div>
-            </article>
-          </div>
-          <div class="c-banner-section c-banner-secondary-section">
-            <div class="row small-up-4">
-              <div class="c-banner-section-row">
-                <div class="small-6 columns">
-                  <a href="http://www.pirati.cz/volby/2017/pardubicko" class="c-card c-card--border">
-                    <div class="c-card-header">
-                      <div class="c-card__title"> Kandidáti do sněmovny v Pardubickém kraji </div>
-                    </div>
-                    <div class="c-card__icon c-card__icon--pos3">
-                      <i class="fa fa-newspaper-o u-rotate-345" aria-hidden="true"></i>
-                    </div>
-                  </a>
-                </div>
-                <div class="small-6 columns">
-                  <a href="{{ '/program/' | relative_url }}" class="c-card c-card--border">
-                    <div class="c-card-header">
-                      <div class="c-card__title"> Program pro volby do poslanecké sněmovny 2017 </div>
-                    </div>
-                    <div class="c-card__icon c-card__icon--pos2">
-                      <i class="fa fa-newspaper-o u-rotate-345" aria-hidden="true"></i>
-                    </div>
-                  </a>
-                </div>
-              </div>
-              <div class="c-banner-section-row">
-                <div class="small-6 columns">
-			<a href="{{ '/plavba/' | relative_url }}" class="c-card c-card--border">
-                    <div class="c-card-header">
-                      <div class="c-card__title"> Můžete nás potkat </div>
-                    </div>
-                    <div class="c-card__icon c-card__icon--pos4">
-                      <i class="fa fa-ship fa-5x" aria-hidden="true"></i>
-                    </div>
-                  </a>
-                </div>
-		{% comment %}
-                <div class="small-6 columns">
-                  <a href="" class="c-card c-card--border">
-                    <div class="c-card-header">
-                      <div class="c-card__title"> Chrudimské pirátské centrum CPICE </div>
-                    </div>
-                    <div class="c-card__icon c-card__icon--pos4">
-                      <i class="fa fa-newspaper-o u-rotate-345" aria-hidden="true"></i>
-                    </div>
-                  </a>
-                </div>
-		{% endcomment %}
-              </div>
-            </div>
-            <!-- .c-card-sections -->
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>

```

Comment:
Ja bych neodstranoval cely blok, misto toho to zakomentuj (roztahni koment z CPICE), potom jen nahradime odkazy.

BTW mas to nekde nahozene, jak to vypada po tech upravach?